dashboard/preferenceForm.ttl

```diff
@@ -0,0 +1,12 @@
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix solid: <http://www.w3.org/ns/solid/terms#>.
+@prefix ui: <http://www.w3.org/ns/ui#>.
+@prefix : <#>.
+
+:this
+<http://purl.org/dc/elements/1.1/title> "Basic preferences" ;
+a ui:Form ;
+ui:part :categorizeUser;
```

Comment:
The line `ui:part :categorizeUser;` is not needed  as that information is already in the following line.   Don't know why I originally wrote both.

Reply:
Added a commit that fixes that.